metals/src/main/scala/scala/meta/internal/metals/InteractiveSemanticdbs.scala

```diff
@@ -87,8 +87,7 @@ final class InteractiveSemanticdbs(
           if (existingDoc == null || existingDoc.md5 != sha) {
             val compiled = compile(path, text)
             Option(compiled).foreach(doc =>
-              // don't index dependency source files or worksheets, since their definitions are local
-              if (!source.isDependencySource(workspace) && !source.isWorksheet)
+              if (!source.isDependencySource(workspace))
```

Comment:
I removed the condition to start indexing worksheet files, since they can contain symbols from the workspace.

Reply:
Just to make sure I'm following. If you now are using a defined type _in_ a worksheet and search for references of that type, this would mean that the result from the worksheet would also appear right?

Reply:
Yes exactly, though for Scala 2 everything defined in the worksheet is local, so in reality these will not pop up in references.
However, this change causes the refreences to workspace symbols, that are used in worksheets, to show up, which I think is super useful.